skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -232,7 +232,6 @@ In the Teams PowerShell Module version 3.0.0 or later, filtering functionality i
 - UserDirSyncEnabled
 - TeamsWorkLoadPolicy
 - VoiceRoutingPolicy
-- LastName (available in Teams PowerShell Module 4.2.0 and later)
```

Comment:
@isrumnon  - is this not there in 4.2.1? if yes, let's add that instead of full removal